### PR TITLE
changes for 05202021 release

### PR DESCRIPTION
**Pull request by daweim0**
Pre-release chores (changing image tag, docker provider version, release notes).

There were no windows changes since the last release, so no new windows image.

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -11,9 +11,15 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+### 05/20/2021 -
+##### Version microsoft/oms:ciprod0520021 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05202021 (linux)
```
Comment (ganga1980) [inline ReleaseNotes.md:15]: typo, should be "ciprod05202021"

File: ReleaseNotes.md [hunk 2]
```diff
@@ -11,9 +11,15 @@ additional questions or comments.
...
+##### No Windows changes with this release, win-ciprod04222021 still current.
+##### Code change log
+- Telegraf now waits 30 seconds on startup for network connections to complete (Linux only)
+- Change adding telegraf to the liveness probe reverted (Linux only)
 
 ### 05/12/2021 -
-##### Version microsoft/oms:ciprod00512021 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05122021 (linux)
+##### Version microsoft/oms:ciprod0512021 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod05122021 (linux)
```
Comment (ganga1980) [inline ReleaseNotes.md:22]: typo :)

File: ReleaseNotes.md (hunk 1 above)
Comment (daweim0) [reply ReleaseNotes.md]: GAH they never end

**Review by ganga1980 — APPROVED** [review]
lgtm

**Pull request merged**